v2/concordium/types.proto

```diff
@@ -3119,3 +3121,105 @@ message BakerRewardPeriodInfo {
   // Whether the baker is a finalizer or not.
   bool is_finalizer = 6;
 }
+
+// The signature of a 'QuorumCertificate'.
+message QuorumSignature {
+  // The bytes representing the raw aggregate signature.
```

Comment:
You could say here that it'll always be fixed length (is it 48 or 96 bytes?)

Reply:
48